src/slackapp/SETUP.md

```diff
@@ -0,0 +1,35 @@
+## Setup Slack App
```

Comment:
@mjmaurer moved this instructions to its own file. I also reworded some things for clarity, mind taking a quick look to see if anything can be improved?